packages/http-client/src/index.ts

```diff
@@ -0,0 +1,37 @@
+import isoMorphicFetch from "isomorphic-fetch";
```

Comment:
why do you need isophorphic fetch? can just use fetch and set the required node version

Reply:
Because with isomorphic fetch, we could use the server side (uses `node-fetch`) and browser (uses `window.fetch`) without knowing the environment.

Reply:
what I am saying I would rather isntall a polyfill instead of this and then only for node https://www.npmjs.com/package/node-fetch#providing-global-access

in all other environments (browsers, workers) we already have fetch, we should treat it in code like it is always provided